services/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+java_library(
+    name = "services",
+    srcs = glob([
+        "src/main/java/io/grpc/**/*.java",
```

Comment:
This is a little too widely scoped.   While each of the services in are grouped into this directory, they are mostly independent.  For example, we have the following java_libraries internally:

* ProtoReflectionService.java
* ChannelzProtoUtil.java, ChannelzService.java
* binary logging

Splitting up helps silo dependencies and speed up builds.   It also prevents files from accidentally get pulled in, which `**` does.

Reply:
Done. The services package now exposes :binarylog, :channelz, and :reflection java_library targets.